scripting/src/main/java/org/infinispan/scripting/impl/ScriptingManagerImpl.java

```diff
@@ -69,7 +69,7 @@ public void initialize(final EmbeddedCacheManager cacheManager, InternalCacheReg
 
    Cache<String, String> getScriptCache() {
       if (scriptCache == null) {
-         scriptCache = cacheManager.getCache(SCRIPT_CACHE);
+         scriptCache = SecurityActions.getUnwrappedCache(cacheManager.getCache(SCRIPT_CACHE));
```

Comment:
Yes, but since this happens in the server initialization, that's exactly what happens